### ISSUE TITLE
Keep DogeHouse Alive!

### ISSUE THREAD

**Issue by dk-raw** [benawad/dogehouse#2858]
This is of utmost importance!

**Comment by thefakequake** [comment]
Ben has already explained in [this video](https://www.youtube.com/watch?v=I8PkQgPiSq8) why he isn't continuing with DogeHouse.

It simply doesn't make sense to pour all this time and effort into a project that is or will be overshadowed by other platforms with more or less the same features.

**Comment by S-codes14** [comment]
> This is of utmost importance!

I Think it is up to the community on github to keep it alive @QuaKe8782  and @dk-raw . If we share dogehouse with our friends and family I think it will be succesful. And knowing the target audience is also beneficial I think that is why @benawad  was [rejected for y - combinator ](https://www.youtube.com/watch?v=IKc_RH33-uM)

**Comment by thefakequake** [comment]
The project needs to be hosted elsewhere though. Ben said he would keep hosting it until he runs out of credit.

**Comment by okdargy** [comment]
Atleast keep it running, make it so the community can donate to hosting fees!

**Comment by S-codes14** [comment]
> The project needs to be hosted elsewhere though. Ben said he would keep hosting it until he runs out of credit.

It just shows how attached he is, and a great inspiration for people

**Comment by S-codes14** [comment]
> Atleast keep it running, make it so the community can donate to hosting fees!

@dragonismcode absolutely  but before people's money 🤑 gets ruined the app needs to scale first!!!!

**Comment by HotBBQSauce** [comment]
Maybe porting it to stacks or some other form of blockchain?

**Comment by Poshy163** [comment]
Yeah that will work, although you would need permission from ben and someone reliable to host it.

**Comment by AntonyLeons** [comment]
> u would need permission from ben and someone reliable to host it.

 MIT License, you already have permission. Nobody is willing to pay for servers.

> Maybe porting it to stacks or some other form of blockchain?

This project is stuggling, porting it to a new stack changes nothing. And what about blockchain?

**Comment by Poshy163** [comment]
>Nobody is willing to pay for servers.

Do we have even have a base cost of running the server? if so, how much?

**Comment by isaxk** [comment]
> >Nobody is willing to pay for servers.
> 
> Do we have even have a base cost of running the server? if so, how much?

Ben wasn't running his own servers. He was using Google Cloud or AWS to host it.

**Comment by Poshy163** [comment]
> Ben wasn't running his own servers. He was using Google Cloud or AWS to host it.

I understand that, but has he said somewhere how much it costs?

**Comment by tomicleon** [comment]
> I understand that, but has he said somewhere how much it costs?

I think I remember him saying in a video it costs him ~$300 (per month) don't quote me on it though.

**Comment by aaryamvn** [comment]
> > Ben wasn't running his own servers. He was using Google Cloud or AWS to host it.
> 
> I understand that, but has he said somewhere how much it costs?

I think in his interview with Dennis he mentioned it costed 300 bucks monthly

@Poshy163

**Comment by chuddyjoachim** [comment]
> > This is of utmost importance!
> 
> I Think it is up to the community on github to keep it alive @QuaKe8782  and @dk-raw . If we share dogehouse with our friends and family I think it will be succesful. And knowing the target audience is also beneficial I think that is why @benawad  was [rejected for y - combinator ](https://www.youtube.com/watch?v=IKc_RH33-uM)

@s-codes14 that'll be a great way to start.

**Comment by bluethefoxofficial** [comment]
@fullstackslayer @AntonyLeons @Poshy163 @chuddyjoachim @Intermo101 
I have decided to take the source code and reload the project myself i have the resources to do so.

i dont know if this has the server side on it but im gonna restart it myself look on my profile for dogehouse-reloaded.

here is the link to the reloaded project: https://github.com/bluethefoxofficial/dogehouse-Reloaded

together we can keep it alive.

**Comment by aaryamvn** [comment]
> @fullstackslayer @AntonyLeons @Poshy163 @chuddyjoachim @Intermo101 
> I have decided to take the source code and reload the project myself i have the resources to do so.
> 
> i dont know if this has the server side on it but im gonna restart it myself look on my profile for dogehouse-reloaded.
> 
> here is the link to the reloaded project: https://github.com/bluethefoxofficial/dogehouse-Reloaded
> 
> together we can keep it alive.

@bluethefoxofficial if you are interested in reviving dogehouse, I am working on a project management and discovery platform called Hydralite and think it would be a unique venture if we tried to integrate dogehouse into it. We were about to create our own audio room service, but if you are interested we can try and use dogehouse code and give it a whole new face. DM me on discord (abno#7777) so we can continue this conversation.

**Comment by jmoseman01** [comment]
The benefit of doing an open source project like this is that you can use an open source business model.  The only main competitor I see in this space for open source is Jitsi.  The project can provide other projects support in integrating it into commercial applications.  I'm in need of a conferencing tool on MyVindex and would like to use something like this. I know Jitsi is doing jitsi as a service as an online solution that other projects can use. - https://jaas.8x8.vc/#/ this project could do something similar.  Also, school districts and such are already paying for zoom you could help them set up there own dogehouse instances at a lower rate because they don't have to deal with licensing.
